rdmo/projects/utils.py

```diff
@@ -128,3 +132,16 @@ def save_import_tasks(project, tasks):
 def save_import_views(project, views):
     for view in views:
         project.views.add(view)
+
+
+def get_invite_email_project_path(invite) -> str:
+    project_invite_path = reverse('project_join', args=[invite.token])
+    # check if the multisite environment is enabled and if the user has a member role
+    if invite.user is not None and (settings.MULTISITE or invite.user.role.member.exists()):
```

Comment:
I wonder if we need the `invite.user.role.member.exists(` here. My idea was that `settings.MULTISITE` determines if we are in a multi site setup or not and `roles` have a member even when not in multi site.

Reply:
yes thank you, I've removed it now.
Each user has a `role.member`  from the model https://github.com/rdmorganiser/rdmo/blob/dd0a23244adfa9d371463e63e71f68cea35e2eef/rdmo/accounts/models.py#L163